fili-core/src/main/java/com/yahoo/bard/webservice/web/filters/RateLimitFilter.java

```diff
@@ -62,31 +73,9 @@ public void filter(ContainerRequestContext request) throws IOException {
 
         // Determine if we should filter based on URL path
         // TODO: Make this based on resource mapped to or at least more configurable with _not_ strings
+        // TODO What is /test and /data ?
         if (path.startsWith("/v1/data") || path.startsWith("/data") || path.startsWith("/test")) {
-
-            MultivaluedMap<String, String> headers = Utils.headersToLowerCase(request.getHeaders());
-
-            // Pick RequestType
-            RequestType type;
-            if (DataApiRequestTypeIdentifier.isBypass(headers)) {
-                // Bypass requests are unlimited
-                type = RequestType.BYPASS;
-            } else if (
-                    DataApiRequestTypeIdentifier.isCorsPreflight(request.getMethod(), request.getSecurityContext())
-            ) {
-                // CORS Preflight requests are unlimited
-                type = RequestType.BYPASS;
-            } else if (DataApiRequestTypeIdentifier.isUi(headers)) {
-                // UI requests have a different limit
-                type = RequestType.UI;
-            } else {
-                type = RequestType.USER;
-            }
-
-            // Get the token
-            SecurityContext securityContext = request.getSecurityContext();
-            Principal user = securityContext == null ? null : securityContext.getUserPrincipal();
-            RequestToken token = rateLimiter.getToken(type, user);
+            RateLimitRequestToken token = rateLimiter.getToken(request);
```

Comment:
try removing /test and /data options and see if any tests break.
Consider extracting "/v1/data" to a constant as well in case we ever want to generalize this.

Reply:
/test is necessary for tests to run, I couldn't find a simple way around that.